scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -70,14 +80,16 @@ def test_support_alternative_backends(xp, data, f_name_n_args):
     dtype_np = getattr(np, dtype)
     dtype_xp = getattr(xp, dtype)
 
-    elements = dict(min_value=dtype_np(-10), max_value=dtype_np(10),
-                    allow_subnormal=False)
-    args_np = [np.asarray(data.draw(npst.arrays(dtype_np, shape, elements=elements)))
-               for shape in shapes]
+    # # To test the robustness of the alternative backend's implementation,
+    # # use Hypothesis to generate arguments
+    # elements = dict(allow_subnormal=False)  # consider min_value, max_value
+    # args_np = [np.asarray(data.draw(npst.arrays(dtype_np, shape, elements=elements)),
+    #                       dtype=dtype_np)
+    #            for shape in shapes]
 
-    # `torch.asarray(np.asarray(1.))` produces
-    # TypeError: can't convert np.ndarray of type numpy.object_.
-    # So we extract the scalar from 0d arrays.
+    # For CI, be a little more forgiving; just generate normally distributed arguments
+    rng = rng_keeper.get_rng(f_name)
+    args_np = [rng.standard_normal(size=shape, dtype=dtype_np) for shape in shapes]
     args_xp = [xp.asarray(arg[()], dtype=dtype_xp) for arg in args_np]
 
```

Comment:
OK, based on CI results, it looks like we still need an `assume` here per https://github.com/scipy/scipy/pull/20967#discussion_r1641947113.